Sources/SpeziOnboarding/OnboardingDataSource.swift

```diff
@@ -50,11 +50,16 @@ public class OnboardingDataSource: Module, EnvironmentAccessible {
     ///
     /// - Parameter consent: The exported consent form represented as `PDFDocument` that should be added.
     /// - Parameter identifier: A 'String' identifying the consent form as specified in OnboardingConsentView.                
-    public func store(_ consent: PDFDocument, identifier: String) async throws {
+    public func store(_ consent: ConsentDocumentExport) async throws {
```

Comment:
This would technically be a breaking change of the public interface here; I would therefore rather suggest to keep the function definition the old way and rather map it to the new `ConsentDocumentExport` here when needed. Also avoids us doing one async call here.

Reply:
That makes sense, so I changed it to the previous version.
But, just to make sure: This PR already is changing the public interface anyways, before my changes, it was:
``` swift
public func store(_ consent: PDFDocument) async throws {
```

And in previous commits to this PR, I already introduced the **identifier** as new parameter:
``` swift
public func store(_ consent: PDFDocument, identifier: String) async throws {
```

Thus, the interface would currently change anyway, right? To avoid this, we could possibly make the identifier optional and set it to the same default value as the identifier in the [OnboardingConsentView](https://github.com/RealLast/SpeziOnboarding/blob/main/Sources/SpeziOnboarding/OnboardingConsentView.swift#L227). Not sure if this is relevant, just wanted to make sure that we do not overlook something :)

Reply:
Very good point, I missed this! Agree, the identifier should have a default value, if we have that in place we don't even need to make it optional.

Great catch!